tox.ini

```diff
@@ -1,7 +1,12 @@
 [tox]
+min_version = 4.4.4
 envlist = py37,py38,py39,py310,py311,pypy3,coverage,docs,quality
+labels =
+    ci-tests = py{37,38,39,310,311,py3}
 
 [testenv]
+package = wheel
+wheel_build_env = build_wheel
```

Comment:
By default, Tox builds a `.tar.gz` file once...but when pip installs that file into the virtual environment Tox creates for testing, pip will first convert the file to a wheel.

The `package` directive tells Tox to create a wheel instead of a `.tar.gz` file, and the `wheel_build_env` directive tells Tox that each environment should use a hard-coded build environment name (its default is to use a name that depends on the Python version, iirc).

(`"build_wheel"` could be any static name, it just happens that I chose `"build_wheel"`.)